test/bddtests/go.mod

```diff
@@ -9,13 +9,7 @@ go 1.12
 require (
 	github.com/DATA-DOG/godog v0.7.13
 	github.com/fsouza/go-dockerclient v1.3.0
-	github.com/go-openapi/swag v0.19.0
 	github.com/pkg/errors v0.8.1
 	github.com/sirupsen/logrus v1.3.0
-	github.com/stretchr/testify v1.3.0
-	github.com/trustbloc/sidetree-core-go v0.0.0-20191017153620-a489e16494b3
-	github.com/trustbloc/sidetree-mock v0.0.0
-
+	github.com/trustbloc/sidetree-core-go v0.0.0-20191028174235-80e3b92d0da1
```

Comment:
Should we be surprised that the bddtests don't refer to its parent project? @fqutishat @bstasyszyn

Reply:
I don't think so. The API is REST - no need to refer to the code. It still ends up testing the functionality.